### ISSUE TITLE
Facility form has shared name attributes for different components

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4092]
## Area or Component
Facility form

## Quick Summary
There are elements with the same accessible name that serve different purposes. Generally these are the address fields for the facility and ordering provider.

The elements are:
- street1
- street2
- city
- phone
- zip code
- state

## Accessibility Violation
Name: The element's name is missing or incorrect: https://cdc-devtools.dequecloud.com/rule-help/en/aria-name-missing-incorrect

See emma's comment below

## Action Requested
Differentiate the `name` attributes for all the above input fields. (Ordering-provider-street1 and faclity-street1, for instance)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=1346058064&range=2:2

**Comment by emyl3** [comment]
@emmastephenson I think we can close this ticket because I am no longer seeing this as an issue on the facility form. I'm not seeing any usage of `aria-label`s for those text inputs. The only error close to the one described is because of the device dropdowns on that page. Let me know if I am mistaken and I can investigate again!

**Comment by emmastephenson** [comment]
@emyl3 unfortunately I don't think that's the case :/ I miswrote the accessibility violation a bit - the issue isn't about an `aria-label`, but rather about the `name` of the element. There are multiple elements on the page with exactly the same `name`, but different input fields - `phone_number` for both ordering provider and facility phone, for instance. Those names need to be unique.

Let me know if you want to chat about this offline! You're right about the device thing btw, but @georgehager is looking to fix that with #4269.

**Comment by emyl3** [comment]
@emmastephenson Thank yo u for the clarification emma! I updated th eticket a bit! :)
